src/components/AddressInputPanel/index.tsx

```diff
@@ -93,6 +95,8 @@ export default function AddressInputPanel({
 
   const error = Boolean(value.length > 0 && !loading && !address)
 
+  const containerLinkText = t('viewOnBlockExplorer')
```

Comment:
It's a bit odd having the translation on a variable, why not use it directly?

Reply:
This one makes sense. Move `t('viewOnBlockExplorer')` directly to render